src/Components/ConfirmDialog.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
```

Comment:
I think Babel automatically adds this, as it's the standard for module code:
http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code